doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -81,6 +81,9 @@ Matter
   * Support for Wi-Fi Network Diagnostic Cluster (which counts the number of packets received and transmitted on the Wi-Fi interface).
   * Default support for nRF7002 revision B.
   * Specific QR code and onboarding information in the documentation for each :ref:`Matter sample <matter_samples>` and the :ref:`Matter weather station <matter_weather_station_app>`.
+  * Feature for setting the default transmission output power for Matter over Thread devices.
+    By default, the feature sets the power to the maximum available one for all targets:
+    9 dBm for nRF52840, 3 dBm for nRF5340, 20 dBm for FEM devices, and 9 dBm for sleepy devices.
```

Comment:
Did you mean 0dBM for sleepy devices? Btw, I noticed you changed it in the samples. Do you think it would make sense to set it in Kconfig.defaults using `if OPENTHREAD_MTD_SED`?

Reply:
Actually, is this release note needed? The feature is added to Thread, and it's only used in Matter samples, right?

Reply:
Right, I will change the value to 0 dBm for sleepy devices.  Regarding the second question, Yes, the feature was added to Thread, so I can remove that line and leave only a note about default settings for all targets in samples.